rustfmt.toml

```diff
@@ -1,2 +1,2 @@
-wrap_comments = true
+wrap_comments = false
```

Comment:
If I set this to true, the rustfmt will cause compile errors on one of the doctests by messing up the formatting :/